### PR TITLE
Add note about quotation marks around mongodb url

### LINKED ISSUE #4503: Issue with "Express Tutorial Part 7: Deploying to production": (short summary here please)
So here https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/deployment#setting_configuration_variables

In this code:

```
>heroku config:set MONGODB_URI='mongodb+srv://cooluser:coolpassword@cluster0-mbdj7.mongodb.net/local_library?retryWrites=true'
Setting MONGODB_URI and restarting limitless-tor-18923... done, v13
MONGODB_URI: mongodb+srv://cooluser:coolpassword@cluster0-mbdj7.mongodb.net/local_library?retryWrites=true
```
MONGODB_URI should not be set in-between single quotation marks, at least, it did not work for me that way. I had to delete them to make it work. I think it is the way you are supposed to define variables in an .env file, but since I'm new to this stuff, I might be wrong.  

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/server-side/express_nodejs/deployment`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/deployment
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/server-side/express_nodejs/deployment/index.html
* Last commit: https://github.com/mdn/content/commit/4b9d9a8bf3ab3db9f7b1efc89b5a91bbf459fe07
* Document last modified: 2021-03-01T12:15:15.000Z

</details>

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes #4503 , and also tidies up Notes to the standard format.

Note, the need for quotation marks has changed several times - i.e. this is not the first time I've toggled what this says. To fix it this time I have removed the marks (that works on my testing on linux and Windows) but also added a note that you might need to add them on some systems).

### CODE REVIEW

**Comment by hamishwillee** [comment]
@Ryuno-Ki Could you please review this? Essentially it changes tips to notes (getting ready for markdown!) and adds a note that about how you might need to use quotation marks for a particular URL (I have had to change this a couple of times - due to changed "something") so just adding note is best solution.

File: files/en-us/learn/server-side/express_nodejs/deployment/index.html [hunk 1]
```diff
@@ -365,9 +365,9 @@ <h4 id="Set_node_version">Set node version </h4>
 <pre class="brush: json">{
   "name": "express-locallibrary-tutorial",
   "version": "0.0.0",
-<strong>  "engines": {
+  "engines": {
     "node": "12.18.4"
```
Comment (Ryuno-Ki) [inline files/en-us/learn/server-side/express_nodejs/deployment/index.html:369]: Nit: Node 12 is in maintenance mode already: https://nodejs.org/en/about/releases/
That's out of scope for this PR, though.
If you go to update the `engines` field, perhaps add an entry for `npm` or `yarn`, too.

File: files/en-us/learn/server-side/express_nodejs/deployment/index.html [hunk 2]
```diff
@@ -469,11 +469,16 @@ <h3 id="Setting_configuration_variables">Setting configuration variables</h3>
 
 <p>We should also use a separate database for production, setting its URI in the <strong>MONGODB_URI</strong>  environment variable. You can set up a new database and database-user exactly <a href="/en-US/docs/Learn/Server-side/Express_Nodejs/mongoose#setting_up_the_mongodb_database">as we did originally</a>, and get its URI. You can set the URI as shown (obviously, using your own URI!)</p>
```
Comment (Ryuno-Ki) [inline files/en-us/learn/server-side/express_nodejs/deployment/index.html:470]:
```suggestion
<p>We should also use a separate database for production, setting its URI in the <strong>MONGODB_URI</strong> environment variable. You can set up a new database and database-user exactly <a href="/en-US/docs/Learn/Server-side/Express_Nodejs/mongoose#setting_up_the_mongodb_database">as we did originally</a>, and get its URI. You can set the URI as shown (obviously, using your own URI!)</p>
```
LaTeX made me see double spaces :sunglasses:

File: files/en-us/learn/server-side/express_nodejs/deployment/index.html [hunk 3]
```diff
@@ -469,11 +469,16 @@ <h3 id="Setting_configuration_variables">Setting configuration variables</h3>
...
 
-<pre class="brush: bash">&gt;heroku config:set <strong>MONGODB_URI</strong>='mongodb+srv://cooluser:coolpassword@cluster0-mbdj7.mongodb.net/local_library?retryWrites=true'
+<pre class="brush: bash">&gt;heroku config:set <strong>MONGODB_URI</strong>=mongodb+srv://cooluser:coolpassword@cluster0-mbdj7.mongodb.net/local_library?retryWrites=true
```
Comment (Ryuno-Ki) [inline files/en-us/learn/server-side/express_nodejs/deployment/index.html:472]: Have you tested this?
It should work. But quotation marks often help if some character could be interpreted by a shell.

File: files/en-us/learn/server-side/express_nodejs/deployment/index.html [hunk 4]
```diff
@@ -469,11 +469,16 @@ <h3 id="Setting_configuration_variables">Setting configuration variables</h3>
...
 Setting MONGODB_URI and restarting limitless-tor-18923... done, v13
 MONGODB_URI: mongodb+srv://cooluser:coolpassword@cluster0-mbdj7.mongodb.net/local_library?retryWrites=true
 </pre>
 
+<div class="notecard note">
+  <h4>Note</h4>
+  <p>On some operating systems you may need to set the set the URL between single quotation marks (e.g. <code>heroku config:set MONGODB_URI='mongodb+srv://...'</code>).</p>
```
Comment (Ryuno-Ki) [inline files/en-us/learn/server-side/express_nodejs/deployment/index.html:479]: Ah, gotcha.

**Review by Ryuno-Ki — APPROVED** [review]
Opening the preview link reports no errors. I found some nitpicks, but they are out-of-scope for this PR.

File: files/en-us/learn/server-side/express_nodejs/deployment/index.html (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/learn/server-side/express_nodejs/deployment/index.html:369]: Ha! Node iterates much faster than I can. This is the version of node last time I did a full update/iterate of Express. At some point I'll run through the tutorial again and update to latest versions.

File: files/en-us/learn/server-side/express_nodejs/deployment/index.html (hunk 3 above)
Comment (hamishwillee) [reply files/en-us/learn/server-side/express_nodejs/deployment/index.html:472]: I have. For me it works both with and without. Previously I've tested and found it not working on way or the other. That's why I added the note - the behaviour seems to be a bit unpredictable.

**Pull request merged**